### PR TITLE
test: PropertiesModal

### PR DESCRIPTION
**Pull request by yardz**
### SUMMARY
Tests for component PropertiesModal

### CODE REVIEW

File: superset-frontend/src/explore/components/PropertiesModal/PropertiesModal.test.tsx [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+import React from 'react';
+import { Slice } from 'src/types/Chart';
+import { render, screen, waitFor } from 'spec/helpers/testing-library';
+import fetchMock from 'fetch-mock';
+import userEvent from '@testing-library/user-event';
+import PropertiesModal from '.';
+
+const createProps = () => ({
+  slice: ({
```
Comment (michael-s-molina) [inline superset-frontend/src/explore/components/PropertiesModal/PropertiesModal.test.tsx:28]: Can you check if you can use some of the mocks available in `spec/fixtures`? I know that we have some slices and charts already defined there. If something is missing we could increment. We can also use these mocks in `fetch` results.

Comment (yardz) [reply superset-frontend/src/explore/components/PropertiesModal/PropertiesModal.test.tsx:28]: I tried to apply the slice but it didn't work, at least not in a "simple" way...

File: superset-frontend/src/explore/components/PropertiesModal/PropertiesModal.test.tsx [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+  render(<PropertiesModal {...props} />);
+
+  await waitFor(() => {
+    expect(
+      screen.getByRole('dialog', { name: 'Edit Chart Properties' }),
+    ).toBeVisible();
+  });
+});
+
+test('Should have modal header', async () => {
```
[the remark this answers is not in the document]
Comment (yardz) [reply superset-frontend/src/explore/components/PropertiesModal/PropertiesModal.test.tsx:183]: Done

**Review by rusackas — APPROVED** [review]
LGTM

**Pull request merged**